js/menu.js

```diff
@@ -64,16 +64,19 @@ function init() {
     });
   });
 
-  $('[data-fl-toggle-menu]').on('click keydown', function(event) {
+  $('.nav-right.focus-outline').on('click keydown', function(event) {
```

Comment:
Can we limit this to the menu, e.g.

```suggestion
  $menuElement.find('.nav-right.focus-outline').on('click keydown', function(event) {
```

Reply:
@tonytlwu updated code.